### PR TITLE
Add static information pages

### PR DESCRIPTION
**Pull request by aurelaukperaj**

### CODE REVIEW

File: pages/templates/pages/about_us.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block content%}
+<h1> TRAVEL AGENCY </h1>
```
Comment (ArlindUkperaj) [inline pages/templates/pages/about_us.html]: Maybe it is better if it is in title case

File: pages/templates/pages/contact_us.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{%block content%}
```
Comment (ArlindUkperaj) [inline pages/templates/pages/contact_us.html]: style guide says to leave a space

File: templates/base.html [hunk 3]
```diff
@@ -16,8 +16,8 @@
         <nav>
             <ul class= 'nav__links'>
                 <li><a href = '/'> Homepage </a></li>
-                <li><a href = '/about'> About </a></li>
-                <li><a href = '/contact_us'> Contact us </a></li>
+                <li><a href = 'about/'> About </a></li>
```
Comment (ArlindUkperaj) [inline templates/base.html]: use `url` tag to generate urls
e.g.
```
<a href="{% url 'about' %}">About</a>
```

**Review by ArlindUkperaj — CHANGES_REQUESTED** [review]
fix issues

File: pages/templates/pages/about_us.html (hunk 1 above)
Comment (aurelaukperaj) [reply pages/templates/pages/about_us.html]: done

File: templates/base.html (hunk 3 above)
Comment (aurelaukperaj) [reply templates/base.html]: yes

File: pages/templates/pages/contact_us.html (hunk 2 above)
Comment (aurelaukperaj) [reply pages/templates/pages/contact_us.html]: ok

**Review by ArlindUkperaj — APPROVED** [review]
seems ok

**Pull request merged**